### ISSUE TITLE
Showing the amount of characters already typed.

### ISSUE THREAD

**Issue by ArnoBuschmann** [icebob/vue-form-generator#233]
Thanks for creating the fom generator, it's great! Short question:

Imagine a textarea with a hint of "Max 500 characters". What would the best way to show, how many of these characters are already typed/used, for example "450 of max 500 characters used"?

**Comment by lionel-bijaoui** [comment]
This is a very good question !
You should look into [custom validator](https://icebob.gitbooks.io/vueformgenerator/validation/custom-validators.html). Maybe you could trigger validation of this field on `@input` (I'm not sure how to do that).
I'm aware that this is not a good solution but right now `hint` is a simple string.
It would be nice to have a dynamic `hint` (like with `visible` or `disabled`).
If you can help us on that, it would be very nice of you to do a PR.
Good luck ! 👍

**Comment by icebob** [comment]
Hint can be function after #303
